outputter/outputter.go

```diff
@@ -17,22 +17,35 @@ func (f outputFunc) Output(controls *check.Controls, summary check.Summary) erro
 	return f(controls, summary)
 }
 
+// Format of the output
+type Format int
+
+const (
+	// JSONFormat send json output to console
+	JSONFormat Format = iota + 10
+	// PgSQLFormat send output to pgsql DB
+	PgSQLFormat
+	// JUnitFormat send JUnit output to the console
+	JUnitFormat
+	// ConsoleFormat send output console
+	ConsoleFormat
+)
+
 // Config configuration for either JSON or Console outputter
 type Config struct {
 	Console
-	JSONFormat  bool
-	JUnitFormat bool
-	Filename    string
+	Format   Format
+	Filename string
 }
 
 // BuildOutputter build new outputter. Depending on the parameters
 // passed will return either a JSON outputter or a Console outputter.
```

Comment:
```suggestion
// BuildOutputter builds a new outputter
```
Let's remove the bit of comment that is outdated. The code is clear,  I don't think it needs further explanation.